cmd/dcrtime/dcrtime.go

```diff
@@ -34,10 +35,12 @@ var (
 	printJson = flag.Bool("json", false, "Print JSON response from server")
 	fileOnly  = flag.Bool("file", false, "Treat digests and timestamps "+
 		"as file names")
-	host    = flag.String("h", "", "Timestamping host")
-	trial   = flag.Bool("t", false, "Trial run, don't contact server")
-	verbose = flag.Bool("v", false, "Verbose")
-	digest  = flag.String("digest", "", "Submit a raw 256 bit digest to anchor")
+	host       = flag.String("h", "", "Timestamping host")
+	trial      = flag.Bool("t", false, "Trial run, don't contact server")
+	verbose    = flag.Bool("v", false, "Verbose")
+	digest     = flag.String("digest", "", "Submit a raw 256 bit digest to anchor")
+	apiVersion = flag.Int("api", 0,
```

Comment:
Shouldn't this default to a valid API version?  I'd suggest the most recent one.